src/content/docs/installation/browser.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Using the nemo playground
+---
+To get started with Nemo without any installation process check out our [interactive browser tool](https://tools.iccl.inf.tu-dresden.de/nemo). It is easy to use and supports code editing: 
+![Nemo web screenshot](../../../assets/nemo-web-screenshot.png)
```

Comment:
`Nemo web screenshot` is a useless alt text. Should be something like “Nemo in the browser, showing the code editor with the ancestors example open on the left, and the execution panel on the right, listing derived facts for a predicate.”

Also, the screenshot should not be cropped in the middle of some words.